### PR TITLE
FF8: more debug output + Fix crash

### PR DESCRIPTION
**Pull request by myst6re**

### CODE REVIEW

File: src/cfg.cpp [hunk 1]
```diff
@@ -217,7 +217,7 @@ void read_cfg()
 	enable_lighting = config["enable_lighting"].value_or(false);
 	prefer_lighting_cpu_calculations = config["prefer_lighting_cpu_calculations"].value_or(true);
 	ff7_external_opening_music = config["ff7_external_opening_music"].value_or(false);
-	ff7_more_debug = config["ff7_more_debug"].value_or(false);
+	more_debug = config["more_debug"].value_or(false) || config["ff7_more_debug"].value_or(false);
```
Comment (julianxhokaxhiu) [inline src/cfg.cpp:220]: Keep only the flag that exists in the `FFNx.toml`. Therefore please remove this or logic, thank you :)

**Review by julianxhokaxhiu — CHANGES_REQUESTED** [review]

File: src/cfg.cpp (hunk 1 above)
Comment (myst6re) [reply src/cfg.cpp:220]: You don't want to keep the compatibility with old config files?

Comment (julianxhokaxhiu) [reply src/cfg.cpp:220]: No, that was never the case so far. If a flag changes, the code changes with it.

Comment (myst6re) [reply src/cfg.cpp]: Okay, as it is not a sensible flag, I can agree with you, but sometimes it is better to keep a compatibility layer for our users :-)

**Review by julianxhokaxhiu — APPROVED** [review]

**Comment by julianxhokaxhiu** [comment]
Thank you :)

**Pull request merged**